CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [0.5.0] - 2023-10-05
```

Comment:
Looks OK to me to be merged if you just update the date to the current one still.